channels/es.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U x-tvg-url="http://195.154.221.171/epg/guidespain.xml.gz"
+#EXTINF:-1 tvg-logo="https://i.imgur.com/zRgQDfM.jpg",PTV Malaga
+http://149.91.98.146:1935/live/stream/playlist.m3u8
```

Comment:
does not work for me
<img width="988" alt="Screenshot 2019-10-20 at 19 57 12" src="https://user-images.githubusercontent.com/7253922/67163122-e927d080-f373-11e9-828c-e385c773bb80.png">